### PR TITLE
Updated code with Styling of Input fields

### PR DESCRIPTION
**Pull request by shaik-amanullah-codelaunch**
Input fields on the Borrow Book page are visibly thicker and easier to interact with.
 Fields are responsive and maintain readability on various screen sizes.
 Visual consistency with the styling of other input elements.

### CODE REVIEW

**Comment by XENONCYBER** [comment]
The width of the input fields are still not proper. give them a max-width and particular width

File: borrow_book.html [hunk 1]
```diff
@@ -220,51 +220,47 @@
...
+        class="input"
+        type="text"
+        id="book-name"
+        name="bookName"
+        placeholder="Enter book Name"
+      />
+      </div>
+      <div class="form-group">
+      <label for="person-identification">Designation</label>
+      <select class="input" id="person-identification" name="designation" style="width: 100%; padding: 10px; border: 1px solid #ccc; border-radius: 4px; margin-top: 5px;">
```
Comment (XENONCYBER) [inline borrow_book.html]: the dropdown text is not visible. it is always white in color. Change the text-color to black

File: borrow_book.html [hunk 2]
```diff
@@ -220,51 +220,47 @@
...
+      <label for="student-usn">USN/ID</label>
+      <input
+        class="input"
+        type="text"
+        id="student-usn"
+        name="usn"
+        placeholder="Enter USN"
+      />
+      </div>
+      <div class="form-group buttons">
```
Comment (XENONCYBER) [inline borrow_book.html:264]: upadte the button div styles the bg is not porperly added

File: borrow_book_styles.css [hunk 3]
```diff
@@ -4,25 +4,28 @@
...
   border-radius: 10px;
-  box-shadow: 1px 1px 4px 4px color(white);
+  box-shadow: 0 4px 30px rgba(0, 0, 0, 0.1);
+  backdrop-filter: blur(10px);
+  -webkit-backdrop-filter: blur(10px);
+  border: 1px solid rgba(255, 255, 255, 0.3);
 }
 
-#data label {
-  margin: 10px;
```
Comment (XENONCYBER) [inline borrow_book_styles.css]: remove unused lines

**Comment by shaik-amanullah-codelaunch** [comment]
Now it's Updated

**Comment by shaik-amanullah-codelaunch** [comment]
Now I have updated the code. Please check it once. Now it's working fine

**Pull request merged**